packages/grid/_modules_/grid/models/gridRows.ts

```diff
@@ -80,7 +80,16 @@ export type GridRowsLookup = Record<GridRowId, GridRowModel>;
  */
 export type GridRowId = string | number;
 
-export type GridRowEntry = { id: GridRowId; model: GridRowModel };
+export interface GridRowEntry {
+  /**
+   * The row model of the row that the current cell belongs to.
```

Comment:
We are using this interface in a lot of selectors that are not related to cells.
Could we rephrase these descriptions to be more generic ?